src/components/Offers/Edit/EditOfferForm.js

```diff
@@ -41,9 +43,18 @@ const parseOfferForm = ({
     vacancies: vacancies || "",
     description,
     descriptionText: parseDescription(description),
+    applyURL: /^mailto:/.test(applyURL) ? applyURL.substring(7) : applyURL,
```

Comment:
Why do we need to remove `mailto:` when we check if it's in the string in `pareApplyURL`? Maybe I missed something, I'm not sure

Reply:
It's not in `parseApplyURL`, it's to ignore `mailto:` when we edit an existing offer. I think if a company that didn't use `mailto:`in the one click application's email, it would be weird to suddenly appear when editing the offer in the future. But yes, it's not mandatory.
![Screenshot from 2023-02-03 13-06-15](https://user-images.githubusercontent.com/42045371/217480092-dada53be-7c02-4b34-a89e-09294a1c774b.png)

Reply:
This is fine but is something that is very rarely going to happen: companies do not need and should not need to know that `mailto:` is a thing, they should just submit their email and it "magically" works. Nevertheless this already catches those rare few that could in fact write that.

Reply:
I think you're talking about the `parseApplyURL` function. This is while loading the offer in the editing page. When the email is fetched, it comes with `mailto:` prepended, and the line mentioned removes `mailto:`.

Reply:
That's the thing: `mailto:` should never be fetched, since its a browser thing. It should just be prepended on the client and not stored on the database, in my opinion.

Reply:
As we discussed in person, mailto is kept on the database because the value could either be en email or a URL. I understand we don't want to show it on the form, so I think this implementation is fine